src/server/pachyderm_test.go

```diff
@@ -1785,11 +1785,14 @@ func TestProvenance(t *testing.T) {
 }
 
 // TestProvenance2 tests the following DAG:
-//   A
-//  / \
```

Comment:
revert please

Reply:
I'm going  to wrap in `/* .. */` so that I don't have to worry about it anymore. I don't actually know how to configure my autoformatter to not format the `//`